app/javascript/hooks/specs/useCurrentUser/FirstCallFailed.spec.tsx

```diff
@@ -58,11 +62,11 @@ describe('useCurrentUser', () => {
 				expect(result.current.error).toBeInstanceOf(NetworkError);
 			});
 
-			it('is not validatingCurrentUser', () => {
+			it('is validatingCurrentUser', () => {
 				expect.assertions(1);
 
 
-				expect(result.current.validatingCurrentUser).toBe(false);
+				expect(result.current.validatingCurrentUser).toBe(true);
```

Comment:
The SWRConfig automatically revalidates on mounting so this will always be validating on mount.